src/panels/config/backup/ha-config-backup-location.ts

```diff
@@ -92,7 +92,7 @@ class HaConfigBackupDetails extends LitElement {
                 </ha-alert>
               `
             : !this.agentId
-              ? html`<ha-circular-progress active></ha-circular-progress>`
+              ? html`<ha-spinner-delayed></ha-spinner-delayed>`
```

Comment:
Why the `delayed` here?

Reply:
Same as in backup details, prevent flickering